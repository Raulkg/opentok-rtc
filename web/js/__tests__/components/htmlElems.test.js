var htmlElems = require('../../components/htmlElems.js');
test('snapshot for htmlElems component', () => {
  expect(htmlElems).toMatchSnapshot();
});