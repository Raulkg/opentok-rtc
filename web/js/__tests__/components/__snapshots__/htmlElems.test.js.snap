// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot for htmlElems component 1`] = `
Object {
  "HTMLElems": Object {
    "addHandlerArchive": [Function],
    "addText": [Function],
    "createElement": [Function],
    "createElementAt": [Function],
    "getAncestorByTagName": [Function],
    "isAction": [Function],
    "replaceText": [Function],
    "setEnabled": [Function],
  },
}
`;
